modules/e2ee/OlmAdapter.js

```diff
@@ -83,6 +83,19 @@ export class OlmAdapter extends Listenable {
         return typeof window.Olm !== 'undefined';
     }
 
+    /**
+     * Updates the current participant key and distributes it to all participants in the conference
+     * by sending a key-info message.
+     *
+     * @param {Uint8Array|boolean} key - The new key.
+     * @retrns {number}
```

Comment:
```suggestion
     * @returns {number}
```
(why is there no linting rule for such things!)